From 0196ca430d1bbbbbf45cd94b7e2c1ddde985df2b Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@st.com>
Date: Mon, 22 Jan 2024 15:43:26 +0100
Subject: [PATCH 0582/1695] dts: stm32: VDDGPU no more always-on for
 stm32mp257f-ev1

Remove always-on on VDDGPU from PMIC in order
to reduce power consumption.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git 73eabf566929c7e2bfccebf6e97563bd396a98ea dts: stm32: VDDGPU no more always-on for stm32mp257f-ev1]
Change-Id: Ieda9eff8cb84f1cae099773fdbb13dbba4082f1e
Signed-off-by: Pascal Paillet <p.paillet@st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/353316
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
---
 core/arch/arm/dts/stm32mp257f-ev1.dts | 1 -
 1 file changed, 1 deletion(-)

diff --git a/core/arch/arm/dts/stm32mp257f-ev1.dts b/core/arch/arm/dts/stm32mp257f-ev1.dts
index 6cb066677..a68d4ffd3 100644
--- a/core/arch/arm/dts/stm32mp257f-ev1.dts
+++ b/core/arch/arm/dts/stm32mp257f-ev1.dts
@@ -116,7 +116,6 @@
 				regulator-name = "vddgpu_pmic";
 				regulator-min-microvolt = <800000>;
 				regulator-max-microvolt = <900000>;
-				regulator-always-on;
 				regulator-over-current-protection;
 			};
 			vddio_pmic: buck4 {
-- 
2.47.2

